src/v/kafka/protocol/schemata/generator.bzl

```diff
@@ -0,0 +1,103 @@
+"""
+This module contains functions for working with Kafka messages.
+"""
+
+load("//bazel:build.bzl", "redpanda_cc_library")
+
```

Comment:
Can we add `visibility("private")` here (the default is public)?

See: https://bazel.build/concepts/visibility#load-visibility

Reply:
Ah probably needs to be `src/v/protocol/...` actually

Reply:
are you referring to the libraries being created by the macro, or this macro .bzl file itself? i'll fix this up in the next pr

Reply:
the .bzl file

Reply:
cool. added it in https://github.com/redpanda-data/redpanda/pull/22672